packages/cli/src/models/network/NetworkController.ts

```diff
@@ -250,14 +250,17 @@ export default class NetworkController {
     await this._setSolidityLibs(libClass); // Libraries may depend on other libraries themselves
     Loggy.spin(__filename, '_uploadSolidityLib', `upload-solidity-lib${libName}`, `Uploading ${libName} library`);
 
-    if (this.project !== undefined) {
-      const libInstance = await this.project.setImplementation(libClass, libName);
-      this.networkFile.addSolidityLib(libName, libInstance);
+    let libInstance;
+
+    if (this.project === undefined) {
```

Comment:
Why not a ternary operator, to avoid the use of `let`?

```ts
const libInstance = (this.project === undefined) ? await Transactions.deployContract(libClass) : await this.project.setImplementation(libClass, libName);
```

Reply:
I don't know, I was influenced by a recent post by Dan Abramov to not be afraid of `let`.

https://overreacted.io/on-let-vs-const

Reply:
Changed to ternary operator.